documentation/pwa/tutorial-defer-endpoint-request.asciidoc

```diff
@@ -48,4 +48,6 @@ if (deferrableResult.isDeferred) {
     const result = deferrableResult.result;
     // your online logic
 }
----- 
+----
+
+For a deferred call, the result is not available in the returned `Deferrable` object. See the <<tutorial-handle-deferredcall-result#, Handle Deferred Call Result>> tutorial on how to handle deferred call results using `onDeferredCall` callback.
```

Comment:
Should be `DeferrableResult`. `Deferrable` was a typo and has been fixed in my previous PR.